### PR TITLE
removes dependencys from other repos, removes bad features, makes it work, update to 144

### PR DESCRIPTION
**Pull request by bend-n**

### CODE REVIEW

**Comment by bend-n** [comment]
@nilederg can we try it?

**Comment by nilederg** [comment]
hmmm

File: plugin.json
Comment (nilederg) [inline plugin.json:1]: better to credit to "ACP Developers"

**Review by nilederg — CHANGES_REQUESTED** [review]
Please change credit to the more general "ACP Developers". Fixing the rest of the formatting can happen in a different PR. Thanks for the work removing the excess code and bloat. That plus the reformatting makes it look a lot nicer.

**Review by nilederg — APPROVED** [review]
Thanks for removing the bloat. LGTM.

**Comment by nilederg** [comment]
I'll leave the merge to you.

**Pull request closed without merging**